src/main/java/command/DeadlineCommand.java

```diff
@@ -0,0 +1,24 @@
+package command;
+
+import main.Ui;
+import task.Deadline;
+import main.Storage;
+import main.TaskList;
+
+import java.time.LocalDateTime;
+
+public class DeadlineCommand extends Command {
+    private final String description;
+    private final LocalDateTime deadline;
```

Comment:
your variable name supposed to be in all caps la bodoh, its a constant!!! never read coding standard tsk..

Reply:
bro static final then is constant lah stupid, i still need to initialise this variable in the constructor